assets/app/views/browse/build-config.html

```diff
@@ -28,6 +28,9 @@
                 ><i class="fa fa-play fa-fw" aria-hidden="true"></i> Start Build</a>
               </li>
               <li>
+                <a ng-href="{{buildConfig | editResourceURL}}" role="button"><i class="fa fa-pencil fa-fw" aria-hidden="true"></i> Edit </a>
```

Comment:
If we change the "Edit (Raw)" to "Edit YAML", the difference might be clearer. @jwforres ?